app/src/main/java/com/babylon/wallet/android/domain/model/messages/RemoteEntityID.kt

```diff
@@ -0,0 +1,19 @@
+package com.babylon.wallet.android.domain.model.messages
+
+sealed interface RemoteEntityID {
+
+    val value: String
```

Comment:
Why not being the `id`? All child classes can just override it with `data class Child(override val id: String,...)`